lms/static/sass/shared/_header.scss

```diff
@@ -19,6 +19,15 @@ header.global {
     min-width: 760px;
   }
 
+  .wrapper-header {
+    @include clearfix();
+    height: 40px;
+    margin: 0 auto;
+    padding: 18px ($baseline/2) 0;
+    max-width: grid-width(12);
```

Comment:
I don't have the context for this work, but do we need to be adjusting the grid here (and on line 361 below) for accessibility?